packages/docs/components/page-configs/va-toast/api-options.ts

```diff
@@ -0,0 +1,23 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
```

Comment:
@untael would be greate if we can implement it in vue 3, as I'm pretty sure there will be issues.
Also I might have skipped a bug or two, so ~~keep your eyes open~~ remember to wipe your space helmet.